integration/stacks_test.go

```diff
@@ -140,15 +143,123 @@ func deployStack(ctx context.Context, oktetoPath, stackPath string) error {
 	return nil
 }
 
-func destroyStack(ctx context.Context, oktetoPath, stackManifest string) error {
+func destroyStack(ctx context.Context, oktetoPath, stackManifest, dir string) error {
 	log.Printf("okteto stack destroy")
 	cmd := exec.Command(oktetoPath, "stack", "destroy", "-f", stackManifest)
 	cmd.Env = os.Environ()
-	cmd.Dir = stackGitFolder
+	cmd.Dir = dir
 	o, err := cmd.CombinedOutput()
 	if err != nil {
 		return fmt.Errorf("okteto stack destroy failed: %s - %s", string(o), err)
 	}
 	log.Printf("okteto stack destroy success")
 	return nil
 }
+
+func TestCompose(t *testing.T) {
+	if mode == "client" {
+		t.Skip("this test is not required for client-side translation")
+		return
+	}
+
+	ctx := context.Background()
+	oktetoPath, err := getOktetoPath(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tName := fmt.Sprintf("TestStacks-%s", runtime.GOOS)
+	name := strings.ToLower(fmt.Sprintf("%s-%d", tName, time.Now().Unix()))
+	namespace := fmt.Sprintf("%s-%s", name, user)
+
+	t.Run(tName, func(t *testing.T) {
+		log.Printf("running %s \n", tName)
+		k8Client.Reset()
+	})
+	if err := createNamespace(ctx, oktetoPath, namespace); err != nil {
+		t.Fatal(err)
+	}
+
+	log.Printf("created namespace %s \n", namespace)
+
+	if err := cloneGitRepo(ctx, composeGitRepo); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := deployStack(ctx, oktetoPath, "docker-compose.yml", composeGitFolder); err != nil {
+		t.Fatal(err)
+	}
+	defer deleteGitRepo(ctx, composeGitFolder)
+
+	log.Printf("deployed stack using %s \n", "docker-compose.yml")
+
+	endpoint := fmt.Sprintf("https://nginx-%s.cloud.okteto.net/db", namespace)
```

Comment:
could we make the url parametric, so we can also run these tests on our dev environments?